docs/reference/developer/concepts.ipynb

```diff
@@ -140,12 +147,17 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {},
+   "metadata": {
+    "tags": []
+   },
    "outputs": [],
    "source": [
-    "a = da['x', 0]\n",
-    "b = da['x', 1]\n",
-    "c = da['x', 2]\n",
+    "a = da['x', 0].copy()\n",
+    "a.attrs['x'] = a.coords.pop('x')\n",
+    "b = da['x', 1].copy()\n",
+    "b.attrs['x'] = b.coords.pop('x')\n",
+    "c = da['x', 2].copy()\n",
+    "c.attrs['x'] = c.coords.pop('x')\n",
     "assert sc.identical(a + (b + c), (a + b) + c)"
```

Comment:
Don't we want to test this concept with `aligned` flag now?

Reply:
With 3 unaligned coords? I think this would fail. We would need to change it so that unaligned coords are dropped when one operand has no coord with the same name.

Reply:
Isn't that what we want? We just want to change `attrs` -> unaligned coord, with no behavior change, don't we?

Reply:
Fixed it. I also rewrote the tests in `data_array_arithmetic_test.cpp` to more clearly show what cases they cover. (And now cover all)